eduNEXT/edunext-platform
# feat(FEC-24): Add context to certificate webview. (DS-598)

# Allow Instructors to manage certificates

## Description

Allow Course Admins ("instructors") of the course access to all the actions available in certificates tab.
Remove css that hides the signatures.

## Supporting information
Last pull request https://github.com/eduNEXT/edunext-platform/pull/689 
[Jira card](https://edunext.atlassian.net/browse/DS-301)

## How to test

1.   Create a course with a certificate
2.  Create a new user without special permissions (superuser, staff).
3.  Add the user to the course team as Admin (i.e. Instructor) via the instructor tab.
4. The certificate tab should be visible and all the certificate operations should work (certificate generation, whitelist, self generation, etc)

For a more specific test description check the [document](https://docs.google.com/document/d/1D9yC98iAcKY6qWd9722Vrfa_Elqgf2NhY1tsT8-TXyE/edit#)
